src/psycopmlutils/loaders/synth/raw/load_synth_data.py

```diff
@@ -0,0 +1,90 @@
+"""Loaders for synth data."""
+
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+
+from psycopmlutils.utils import data_loaders
+
+
+def load_raw_test_csv(filename: str, n_rows: Optional[int] = None) -> pd.DataFrame:
+    """Load raw csv.
+
+    Args:
+        filename (str): Name of the file to load.
+        n_rows (int, optional): Number of rows to load. Defaults to None.
+    """
+    # Get project root dir
+    project_root = Path(__file__).resolve().parents[5]
```

Comment:
```suggestion
    project_root = Path()
```
`Path()` gives you the project root

Reply:
It gives you ".", which is relative to where you are running it from, not from the file.